### PR TITLE
Sphinx documentation files for ReadTheDocs

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: docs/features.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Features
+========
+
+**SNV and InDel support**
+
+pVAC-Seq offers epitope binding predictions for missense, inframe indel, and frameshift mutations.
```
Comment (chrisamiller) [inline docs/features.rst]: Why sometimes "InDel" and sometimes "Indel"?  FWIW, the latter is more common.

File: docs/index.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+.. pVAC-Seq documentation master file, created by
+   sphinx-quickstart on Wed Aug 31 08:31:01 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+pVAC-Seq
+====================================
+
+pVAC-Seq is a cancer immunotherapy pipeline for the identification of **p**\ ersonalized **V**\ ariant **A**\ ntigens by **C**\ ancer **Seq**\ uencing (pVAC-Seq) that integrates tumor mutation and expression data (DNA- and RNA-Seq). It enables cancer immunotherapy research by using massively parallel sequence data to predicting tumor-specific mutant peptides (neoantigens) that can elicit anti-tumor T cell immunity. It is being used in studies of checkpoint therapy response and to identify targets for cancer vaccines and adoptive T cell therapies. For more general information, see the [manuscript published in Genome Medicine](http://www.genomemedicine.com/content/8/1/11)
+
```
Comment (chrisamiller) [inline docs/index.rst]: I'm not sure I did the above link correctly - can someone review and fix, if necessary?

File: pvacseq/lib/coverage_filter.py [hunk 3]
```diff
@@ -4,40 +4,62 @@
...
+    parser.add_argument(
+        '--trna-cov', type=int,
+        help="Tumor RNA Coverage Cutoff. Sites above this cutoff will be considered. "
+             + "Default: 10",
+        default=10
+    )
+    parser.add_argument(
+        '--normal-vaf', type=int,
+        help="Normal VAF Cutoff. Sites BELOW this cutoff in normal will be considered. "
+             + "Default: 2",
```
Comment (chrisamiller) [inline pvacseq/lib/coverage_filter.py]: Any reason why "BELOW" and not just "below"?  (every other above/below is in lowercase)

Comment (susannasiebert) [inline pvacseq/lib/coverage_filter.py]: I think it's to emphasize it since it's different from the other vaf cuttoff behavior.

**Pull request merged**